parlai/core/metrics.py

```diff
@@ -103,13 +112,30 @@ def _bleu(guess, answers):
     )
 
 
+def _rouge(guess, answers):
+    """Compute ROUGE score between guess and *any* answers. Return the best."""
+    if rouge is None:
+        return None, None, None
+    evaluator = rouge.Rouge(metrics=['rouge-n', 'rouge-l'], max_n=2)
```

Comment:
maybe `n` should be an optional parameter, with `n=2` being the default? This is good though; we can change it later.

Reply:
talked offline, decided not to do this.